LR_3/Частотный анализ букв в тексте.py

```diff
@@ -0,0 +1,80 @@
+def count_letters(text):
+    russian_alphabet = [
+        'а', 'б', 'в', 'г', 'д', 'е', 'ё', 'ж', 'з', 'и',
+        'й', 'к', 'л', 'м', 'н', 'о', 'п', 'р', 'с', 'т',
+        'у', 'ф', 'х', 'ц', 'ч', 'ш', 'щ', 'ъ', 'ы', 'ь',
+        'э', 'ю', 'я'
+    ]
+
+    letter_counts = {letter: 0 for letter in russian_alphabet}
+
+    for char in text:
+        if char.isalpha():
+            char = char.lower()
+            if char in letter_counts:
+                letter_counts[char] += 1
+            else:
+                pass
+    return letter_counts

```

Comment:
А здесь необходимо слегка видоизменить логику процесса подсчета. 
Выполним следующую логическую цепь событий:
проверяем что это буква -> приводим в нижний регистр -> проверяем есть ли в словаре букв -> если есть, то увеличиваем значение на +1 -> если нет, то вводим эту букву в словарь и присваиваем ей значение 1
```suggestion
    for char in text:
        if char.isalpha():  #Проверили что символ является буквой
            char = char.lower() #Привели к нижнему регистру
            if char in letter_counts:
                letter_counts[char] += 1 #Если есть в словаре, то увеличиваем значение на единицу
            else:
                letter_counts[char] = 1 #Если нет в словаре, то добавляем со значением 1
    return letter_counts

```